modules/tracking/samples/lasot_benchmark.py

```diff
@@ -0,0 +1,193 @@
+import numpy as np
+import cv2 as cv
+import argparse
+import os
+import glob
+
+# Method used for evaluate corners coords 
+# Not used yet - need to re-write calculations with it
+def calc_coords(bb):
+    xmin = bb[0]
+    xmax = bb[0] + bb[2] - 1.0
+    ymin = bb[1]
+    ymax = bb[1] + bb[3] - 1.0
+    cx = bb[0] + (bb[2] + 1.0) / 2
+    cy = bb[1] + (bb[3] + 1.0) / 2
+    coords = {'xmin': xmin, 'xmax': xmax, 'ymin': ymin, 'ymax': ymax,
+                   'cx': cx, 'cy': cy}
+    return coords
+
+# Method used for initializing of trackers by creating it 
+# via cv.TrackerX_create()
+def init_tracker(tracker_name):
+    config = {"Boosting": (cv.TrackerBoosting_create(), 500),
+    "MIL": (cv.TrackerMIL_create(), 1000),
+    "KCF": (cv.TrackerKCF_create(), 250),
+    "MedianFlow": (cv.TrackerMedianFlow_create(), 500),
+    "GOTURN": (cv.TrackerGOTURN_create(), 250),
+    "MOSSE": (cv.TrackerMOSSE_create(), 250),
+    "CSRT": (cv.TrackerCSRT_create(), 250)}
+    return config[tracker_name]
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Run LaSOT-based benchmark for visual object trackers")
+    # As a default argument used name of 
+    # original dataset folder
+    parser.add_argument("--path_to_dataset", type=str,
+                        default="LaSOTTesting", help="Full path to LaSOT")
+    parser.add_argument("--visualization", type=str,
+                        default=False, help="Showing process of tracking on video")
+    args = parser.parse_args()
+
+    # Creating list with names of videos via reading names from txt file
+
+    video_names = os.path.join(args.path_to_dataset, "testing_set.txt")
+    with open(video_names, 'rt') as f:
+        list_of_videos = f.read().rstrip('\n').split('\n')
+
+    trackers = [
+        'Boosting', 'MIL', 'KCF', 'MedianFlow', 'GOTURN', 'MOSSE', 'CSRT']
+
+    # Loop for every tracker
+    for tracker_id in trackers:
+
+        tracker_name = tracker_id
+        print(tracker_name)
+
+        # Loop for every video
+        for video_name in list_of_videos:
+
+            tracker, frames_before_reinit = init_tracker(tracker_name)
+            init_once = False
+
+            print("Video: " + str(video_name))
+
+            # Open specific video and read ground truth for it
+            gt_file = open(os.path.join(args.path_to_dataset, video_name, "groundtruth.txt"), "r")
+            gt_bb = gt_file.readline().replace("\n", "").split(",")
+            print(gt_bb)
+            init_bb = gt_bb
+            init_bb = tuple([float(b) for b in init_bb])
+
+            print("Initial bounding box: ", init_bb)
+
+            # Creating blob from image sequence
+            #video_sequence = sorted(os.listdir(os.path.join(args.path_to_dataset, video_name, "img")))
+            video_sequence = sorted(glob.glob(str("D:/lasot/" + str(video_name) +
+                                                  "/img/*.jpg")))
```

Comment:
I think it's better to use `os.listdir` because we can delete `glob`. `os.listdir` gives file names from directory. And below you can use:

```python
for number_of_the_frame, image in enumerate(video_sequence):
    frame = cv.imread(os.path.join(args.path_to_dataset, video_name, image))
```

Reply:
Maybe I am doing something wrong, but `os.path.join` does not work here.
When I use `glob`, it creates the list of `D:\lasot\airplane-1\img\00000001.jpg`, and os.path.join creates list of `00000001.jpg`, so the script can not find any images.

Reply:
Yes, `os.path.join` creates list of *.jpg. Below you can change 
`frame = cv.imread(image)`
to 
`frame = cv.imread(os.path.join(args.path_to_dataset, video_name, image))`

Or do you have another question?

Reply:
I think I found the mistake:
it should be `frame = cv.imread(os.path.join(args.path_to_dataset, video_name, "img", image))`.
I missed the name of the folder with pictures.
I will test it a few more times and add it to the next commit.

Reply:
As I see  - it works. I remove `glob` and using `os.path.join` now.